xnmt/input_reader.py

```diff
@@ -198,12 +198,13 @@ class H5Reader(InputReader, Serializable):
   """
   yaml_tag = u"!H5Reader"
 
-  def __init__(self, transpose=False, feat_from=None, feat_to=None, feat_skip=None, word_skip=None):
+  def __init__(self, transpose=False, feat_from=None, feat_to=None, feat_skip=None, word_skip=None, word_truncate=None):
```

Comment:
Because I'm not familiar with this class I'm not sure what this PR is doing. Could you document the `__init__` function?